libnd4j/include/array/DataTypeConversions.h

```diff
@@ -36,33 +36,36 @@ namespace sd {
 template <typename T>
 class SD_LIB_EXPORT DataTypeConversions {
  private:
+
+  // Here is your rewritten rconv function.
```

Comment:
This method looks a bit suspicious. If everything is as it should be, remove the comments that suggest that things may need to be done differently.